lk2nd/device/dts/msm8952/msm8917-xiaomi-rolex.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8917 0>, <308 0>, <309 0>;
+	qcom,board-id = <0x1000b 1>, <0x2000b 1>;
+};
+
+&lk2nd {
+	model = "Xiaomi Redmi 4A (rolex)";
+	compatible = "xiaomi,rolex";
+
+	lk2nd,dtb-files = "msm8917-xiaomi-rolex";
+
+	lk2nd,match-panel;
+	panel {
+		compatible = "xiaomi,rolex-panel", "lk2nd,panel";
```

Comment:
#include "dsi-panel-hx8394f-boe-c3a-720p-video.dtsi"
#include "dsi-panel-nt35521s-ebbg-c3a-720p-video.dtsi"
#include "dsi-panel-ili9881c-ebbg-c3a-720p-video.dtsi"

It seems rolex only have 3 panel variants.

Reply:
Remove ebbgDJN variant.

Reply:
And there is no tianma panel you should check dsi-panel-nt35521s-ebbg-c3a-720p-video.dtsi instead of it.
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/commit/2dd7cd829e165917bfe384febf21d9727b05fba4#diff-9af61c7ea9db8f0811d139a90e483bf2ec4493188b2513106d2657c14aadf6d4R20

Reply:
It seems none of the node names contains c3a.